.github/workflows/main.yml

```diff
@@ -59,7 +59,7 @@ jobs:
   typos:
     env:
       TASK: check-typos
-      TOOLCHAIN: 1.74.1  # CURRENT DEVELOPMENT RUST TOOLCHAIN
+      TOOLCHAIN: 1.75.0  # CURRENT DEVELOPMENT RUST TOOLCHAIN
```

Comment:
Insert 

```
      - uses: dtolnay/rust-toolchain@master
        with:
          toolchain: ${{ env.TOOLCHAIN }}
```

some where before the ```cargo-install``` rule and this should work properly.

Reply:
OK; looks logical, considering the other jobs are using it.